Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 authors = ["NikVolf <nikvolf@gmail.com>"]
 
 [dependencies]
-pwasm-std = { git = "https://github.com/paritytech/pwasm-std" }
+pwasm-std = { git = "https://github.com/paritytech/pwasm-std", rev = "659b810d" }
```

Comment:
No need to reference revision in toml, use latest
once it's on crates.io, we'll use semantic versionning

Reply:
Yeah, but actually tests doesn't work with the latest version of `pwasm-std` because of `mem*` externalization.

Reply:
oh but they should?

Reply:
Not really, yet. I'm making this test in order to fix `_emscripten_memcpy_big` issue. After it lands, I will continue my work on implementing that `mem*` stuff. At least that was my plan.